install/charts/templates/deployment.yaml

```diff
@@ -22,7 +22,7 @@ metadata:
     app: {{ .Values.metadata.labelApp }}
   namespace: {{ .Release.Namespace }}
 spec:
-  replicas: 1
+  replicas: 3
```

Comment:
I would make this configurable in the Helm chart.

Reply:
Done.